### ISSUE TITLE
Table of Contents header contrast too low in dark mode

### ISSUE THREAD

**Issue by nigelmegitt** [w3c/tr-design#354]
In dark mode the variable `--tocsidebar-heading-text` doesn't have a color resulting in enough contrast for the TABLE OF CONTENTS header text:

![image](https://github.com/w3c/tr-design/assets/4988915/3fb731ec-0cce-47ab-a032-6bbc24e641ec)

This variable seems to be used in `base.css` - I guess there isn't a dark mode setting for this variable at all, or it's set to the wrong color. Sorry I didn't have time to dig further!
